index.html

```diff
@@ -134,9 +77,179 @@ <h1>I’m Felix<br />Glad to see you!</h1>
             <img src="images/header-shapes mobile.png" alt="header-shapes" />
           </div>
         </section>
-
+        </section>
         <!-- hero section ends-->
-      </div>
-    </header>
+
+        <!-- Portfolio section starts-->
+        <section id="portfolio-section">
+          <div class="grid-container">
+
+            <div class="card">
+             <a href=""><img src="images/Snapshoot Portfolio.png" alt=""></a>
+             <a href="#" class="work-title"><h2>Tonic</h2></a> 
+            <div class="work-details">
+              <h4 class="sub-title">CANOPY</h4>
+              <span class="bullet-point"></span>
+              <h3>Back End Dev</h3>
+              <span class="bullet-point"></span>
+               <h3>2015</h3>
+              </div>
+              <p>A daily selection of privately personalized reads; no accounts or<br>sign-ups required.</p>
+              <div class="work-tools">
+                <button type="button" class="tools-btn">html</button>
+                  <button type="button" class="tools-btn">css</button>
+                  <button type="button" class="tools-btn">JavaScript</button>
+              </div>
+              <button type="button" class="primary-btn">See Project</button>
+            </div>
+
+             <div class="card">
+             <a href=""><img src="images/Snapshoot Portfolio (1).png" alt=""></a>
+             <a href="#" class="work-title"><h2>Multi-Post<br>
+                Stories</h2></a> 
+            <div class="work-details">
+              <h4 class="sub-title">CANOPY</h4>
+              <span class="bullet-point"></span>
+              <h3>Back End Dev</h3>
+              <span class="bullet-point"></span>
+               <h3>2015</h3>
+              </div>
+              <p>A daily selection of privately personalized reads; no accounts or<br>sign-ups required.</p>
+              <div class="work-tools">
+                <button type="button" class="tools-btn">html</button>
+                  <button type="button" class="tools-btn">css</button>
+                  <button type="button" class="tools-btn">JavaScript</button>
+              </div>

```

Comment:
- [ ] Kindly make sure to wrap your stack into the `ul` tag and wrap each one into the `li` tag in order to correctly format your HTML semantincally. :wink:

Reply:
Thanks @levy002